packages/react-dev-utils/formatWebpackMessages.js

```diff
@@ -39,6 +33,12 @@ function formatMessage(message) {
     lines[0] = lines[0].substr(lines[0].lastIndexOf('!') + 1);
```

Comment:
This line would crash if `lines` had no elements.
I think it should be `if (lines[0] && ...`